src/classes/TDTM_Filter_TEST.cls

```diff
@@ -163,7 +163,8 @@ public with sharing class TDTM_Filter_TEST {
     }
     
     public static testmethod void InvalidFilterField() {
-        List<TDTM_Global_API.TdtmToken> tokens = setup();
+        //If we use getDefaultTDTMConfig from setup, then we can't save the token with invalid field. Because even we set  THAN_Filter_TDTM = inacive and save the token (insert TriggerHandler), system will call getTdtmConfig and get it from default config. So, in here, we need to modify the cache directly.
```

Comment:
Sorry @weizsforg, this comment is throwing me off a bit. It says "system will call getTdtmConfig and get it from default config". Shouldn't getTdtmConfig get whatever is in the database? Maybe the comment is outdated?

Reply:
Sorry about the confusion here. yes, getTdtmConfig "should" get whatever is in the database. However, if the database is empty (like in this test case), it will read the default config and return it.

I think I will modify the comments to make it more clear.